client/src/pages/Cart/CartPage.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { render } from 'test-utils/rtkProvider';
+import { waitFor, screen, act, getByRole, findByRole } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { renderAppPage } from '../../test-utils/renderApp';
+import { getCart } from 'store/feature/cart/cartSlice';
+window.alert = jest.fn();
+
+describe('CartPage 테스트', () => {
+  test('장바구니에 추가된 아이템은 장바구니 store에 저장되어있다.', async () => {
```

Comment:
아래처럼 명명해주셔도 좋을거 같네요!

```suggestion
  test('장바구니에 추가된 아이템은 장바구니 store에 저장되어야 한다.', async () => {
```